### PR TITLE
Use key-value logging in trace logs

### PR DESCRIPTION
**Pull request by Thomasdezeeuw**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -39,7 +39,7 @@ os-ext = ["os-poll"]
 net = []
 
 [dependencies]
-log = "0.4.8"
+log = { version = "0.4.16", default-features = false, features = ["kv_unstable"] }
```
Comment (Darksonn) [inline Cargo.toml:42]: This is an unstable feature?

Comment (Thomasdezeeuw) [reply Cargo.toml:42]: Yes, but it's been working on for over a year and it's close to table, also see https://github.com/rust-lang/log/issues/436.

**Comment by Thomasdezeeuw** [comment]
Friendly ping @Darksonn

**Comment by Darksonn** [comment]
I don't think we should depend on an unstable feature in mio, even if it will be stabilized soon.

Even if nothing else changes, surely the feature will not be stabilized under the name "kv_unstable"?

**Comment by Thomasdezeeuw** [comment]
Fair enough, will wait a bit then.

**Pull request closed without merging**